starter-code/database.js

```diff
@@ -54,11 +54,10 @@ class Database {
       if (error){
         callback(error);
       } else {
-        //  LAB 2
-        // Implement the query to insert a user
-        // remeber once it's finish to comment callback('Error listing users');
+
+        database.collection('users').find(callback);
```

Comment:
you should try and call the firstname as well
        **database.collection('users').find(users.firstName, callback);**